include/bluetooth/gatt.h

```diff
@@ -1332,8 +1332,7 @@ struct bt_gatt_read_params {
 	/** Read attribute callback. */
 	bt_gatt_read_func_t func;
 	/** If equals to 1 single.handle and single.offset are used.
-	 *  If >1 Read Multiple Characteristic Values is performed and handles
-	 *  are used.
+	 *  If >1 multiple.handles are used.
```

Comment:
```suggestion
	 *  If greater than 1 multiple.handles are used.
```
or
```suggestion
	 *  If larger than 1 multiple.handles are used.
```

To be consistent with the `equals` on the line before.